new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -175,6 +176,11 @@ class LogFileManager {
             } due to a limitation in Chromium-based browsers.`);
         }
 
+        const result = decoder.buildIdx(0, FULL_RANGE_END_IDX)
+        if (result != null && result.numInvalidEvents > 0) {
```

Comment:
lint:
```suggestion
        if (null !== result && 0 < result.numInvalidEvents) {
```
1. Let's use "yodo" conditions, where we put constants / literals on the left of the comparison operators.
2. In JavaScript, the difference between `!=` and `!==` is that the former does "loose inequality" comparison, where type conversions happen before the values are compared. The later one compares the type as well. Here's a good reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness